DOCKER_HUB.md

```diff
@@ -0,0 +1,49 @@
+# Quick reference
+
+-    **Maintained by**:  
+     [georchestra.org](https://www.georchestra.org/)
+
+-    **Where to get help**:  
+     the [geOrchestra Github repo](https://github.com/georchestra/georchestra), [IRC chat](https://matrix.to/#/#georchestra:osgeo.org), Stack Overflow
+
+# Featured tags
+
+- `latest`
+
+# Quick reference
+
+-	**Where to file issues**:  
+     [https://github.com/georchestra/georchestra/issues](https://github.com/georchestra/georchestra/issues)
+
+-	**Supported architectures**:   
+     [`amd64`](https://hub.docker.com/r/amd64/docker/)
+
+-	**Source of this description**:  
+     [docs repo's directory](https://github.com/georchestra/maelstro/blob/main/DOCKER_HUB.md)
+
+# What are `georchestra/maelstro-backend` and `georchestra/maelstro-frontend`
+
+**Maelstro-backend** and **Maelstro-frontend** are modules for geOrchestra which offers
+- Metadata copy tool from different geonetwork and geoserver instances
```

Comment:
which offer data and metadata synchronisation facilities between geOrchestra environments (like preprod to production).